fdbserver/TLogServer.actor.cpp

```diff
@@ -554,12 +558,14 @@ struct LogData : NonCopyable, public ReferenceCounted<LogData> {
 		queueCommittedVersion.initMetric(LiteralStringRef("TLog.QueueCommittedVersion"), cc.id);
 
 		specialCounter(cc, "Version", [this](){ return this->version.get(); });
-		specialCounter(cc, "QueueCommittedVersion", [this](){ return this->queueCommittedVersion.get(); });
+		specialCounter(cc, "QueueCommittedVersion",
```

Comment:
@sfc-gh-almiller  Would you mind explaining what these counters' definitions are?

Reply:
The disk queue has committed up until version `QueueCommittedVersion`